### PR TITLE
DIT-2754 | view for manage keywords 

### PR DESCRIPTION
**Pull request by warrenskiwork1979**

### CODE REVIEW

**Review by kuldeepwadhwa9 — APPROVED** [review]

File: test/unit/views/managementtools/KeywordsTabSpec.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    "include an edit keyword button" in {
+      val doc = view(
+        manageKeywordsView()
+      )
+      doc.getElementById("keyword_search-edit-keyword-button") should containText("Edit keyword")
+    }
+
+    //TODO Test search box
```
Comment (emilia-trendafilova) [inline test/unit/views/managementtools/KeywordsTabSpec.scala:67]: is this needed?

**Review by emilia-trendafilova — APPROVED** [review]
LGTM

File: app/models/viewmodels/managementtools/Keyword.scala [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.viewmodels.managementtools
+
+import models.CaseStatus.CaseStatus
+import models.{ApplicationType, CaseStatus}
+
+case class Keyword(
```
Comment (emilia-trendafilova) [inline app/models/viewmodels/managementtools/Keyword.scala:22]: Can you check that the model for Keyword matches the one in the backend? For reference : DIT-2760

**Review by emilia-trendafilova — APPROVED** [review]

File: app/models/viewmodels/managementtools/Keyword.scala (hunk 2 above)
Comment (warrenskiwork1979) [reply app/models/viewmodels/managementtools/Keyword.scala:22]: So this is just a stub at the moment for this ticket which will be replaced with the new Keyword model matching the backend in DIT-2755, which is in progress.

**Pull request merged**